src/components/fieldset/_fieldset.scss

```diff
@@ -9,6 +9,16 @@
     padding: 0;
     border: 0;
     @include govuk-h-clearfix;
+
+    .govuk-c-input:last-of-type,
+    .govuk-c-select:last-of-type {
+      margin-bottom: 0;
+    }
+  }
+
+  .govuk-c-fieldset--error {
+    padding-left: $govuk-spacing-scale-3;
+    border-left: $govuk-border-width-error-field-group solid $govuk-red;
```

Comment:
Should be using the semantic variable – `$govuk-error-colour`.

Reply:
Done!